controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaInterceptor.java

```diff
@@ -49,12 +49,14 @@
     public <ReqT, RespT> ServerCall.Listener<ReqT> interceptCall(ServerCall<ReqT, RespT> call, Metadata headers, ServerCallHandler<ReqT, RespT> next) {
 
         Map<String, String> paramMap = new HashMap<>();
-        headers.keys().stream().forEach(key -> {
+        headers.keys().stream()
+               .filter(key -> !key.endsWith(Metadata.BINARY_HEADER_SUFFIX))
+               .forEach(key -> {
             try {
                 paramMap.put(key,
                         headers.get(Metadata.Key.of(key, Metadata.ASCII_STRING_MARSHALLER)));
             } catch (IllegalArgumentException e) {
-                log.warn("Error while marshalling some of the headers", e);
+                log.info("Error while marshalling some of the headers" + e);
```

Comment:
Reducing the log message by removing the stack trace is not necessarily the right thing to do here. If there is an error, then it is best that the user knows about it and has detail. At the same time, I'm worried that we are simply logging a message and moving forward. It is like the error doesn't matter, and if it is the case, then why are we even throwing in the first place?

Reply:
May I suggest the following principle, that planned errors be treated differently from unplanned errors.   A planned error (i.e. an intentional part of program flow) need not have a stack trace.   For example, bad requests from a client are handled by validation logic and summarily rejected; there's no value in a server-side stack trace in that case (since the logic is working as intended).

Reply:
(if you go with Andrei's suggestion, be sure to add `{}` to the message)

Reply:
Fixed.